### PR TITLE
Postgres: Return tables with hyphenated schemes

### LINKED ISSUE #19743: Tables from Postgres datasource that are created under schemas with dashes are not listed
**What happened**:
In the query tool when choosing from 'select table', tables that exists in schema `test-schema` doesnt shows up, even if `search_path` for that user contains `test-schema`.

**What you expected to happen**:
That the tables are shown

**How to reproduce it (as minimally and precisely as possible)**:
Postgres setup
```
CREATE SCHEMA "test-schema";
ALTER ROLE youruser SET search_path = "test-schema","public";
CREATE TABLE "test-schema".test(id bigint);
```
Add postgres as datasource and connect with User: `youruser` and try to create a graph and see if `test`-table is listed in the drop down.

**Anything else we need to know?**:
Its possible to write "test-schema".test in the table-field, its just that it isn't shown.

**Environment**:
- Grafana version: v5.3.4 (commit: 69630b9)
- Data source type & version: Postgresql V11
- OS Grafana is installed on: Kubernetes, with helm, but its the offical docker image `grafana/grafana`
- User OS & Browser: Linux, Firefox 69.0.2
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by zuchka**
**What this PR does / why we need it**:

This issue only impacts the `FROM` select box when using the Postres data source plugin's query builder.

If you have a schema with a hyphen in it (i.e., `test-schema`), any associated tables will not get populated in the query builder. The [linked issue](https://github.com/grafana/grafana/issues/19743) describes the situation well, and the suggested fix 😄 

to test, add a schema with a hyphen, add the schema to your search path, and then add a table to that schema:

```
CREATE SCHEMA "test-schema";
ALTER ROLE youruser SET search_path = "test-schema","public";
CREATE TABLE "test-schema".test(id bigint);
```

as is, the table will not appear in the query builder

**Which issue(s) this PR fixes**:

Fixes #19743

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/postgres/meta_query.test.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { describe, expect } from 'test/lib/common';
+import { PostgresMetaQuery } from './meta_query';
+import QueryModel from './postgres_query_model';
+
+describe('Postgres MetaQuery', () => {
+  describe('buildSchemaConstraint()', () => {
+    describe('when executing buildSchemaConstraint()', () => {
+      const postgresMetaQuery = new PostgresMetaQuery({ table: '', timeColumn: '' }, new QueryModel({}));
+      it('should return a schema restraint', () => {
+        expect(postgresMetaQuery.buildSchemaConstraint()).toEqual(
```
Comment (zoltanbedi) [inline public/app/plugins/datasource/postgres/meta_query.test.ts:10]: I'm not quite sure about this test. I feel like that it doesn't really test anything. Maybe add a test like this.
`should wrap table_schema with quote_ident` or just add a comment to the buildSchemaConstraint function that explains why we added the quote_ident function.

**Review by zoltanbedi — APPROVED** [review]
Thanks for fixing these issues @zuchka. I've tried this out and it works well.

File: public/app/plugins/datasource/postgres/meta_query.test.ts (hunk 1 above)
Comment (zuchka) [reply public/app/plugins/datasource/postgres/meta_query.test.ts]: good call! yeah, after I wrote that test I realized it didn't really test much. Swapped it out for an inline comment 👍

Comment (zoltanbedi) [reply public/app/plugins/datasource/postgres/meta_query.test.ts]: Awesome, thank you!

**Comment by thall** [comment]
Ohh that old issue! Thanks! :smile:

**Pull request merged**